src/main/java/Parser.java

```diff
@@ -0,0 +1,132 @@
+public class Parser {
+
+    public static final int BEGIN_DEADLINE_INDEX = 9;
+    public static final int BEGIN_EVENT_INDEX = 6;
+
+    public static Command parse(String input, TaskList taskList) throws DukeException {
+        String firstWord = input.split(" ")[0];
+
+        switch (firstWord) {
+        case "bye": {
+            return new ByeCommand();
+        }
```

Comment:
Should the case statement have braces? I noticed the same issue in several other places too.
```suggestion
        case "bye":
            return new ByeCommand();
```